walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,18 +15,31 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Arrays;
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    //no attendees
+    
+    //initialize availibility collection
+    Collection<TimeRange> availibility;
+    
+    
+    //no attendees -- first test
     if (events == null){
-        return 0;
+        availibility = Arrays.asList(TimeRange.WHOLE_DAY);
     }
 
-    //no options too long of a request (how do I access time range here? THis is an error)
-    else if (TimeRange > 25){
-        return 0;
+    //no options too long of a request (how do I access time range here? THis is an error) -- second test
+    else if (request.duration > 24){
+        //availibility = Arrays.asList();
     }
 
+    //event splits day into two options (before and after) -- third test
+    //Should this be in a check or just always happen? 
```

Comment:
For the 2 option split check in the test file (FindMeetingQueryTest.java starting line 82), should I use an if statement to check if a certain condition is met before I assign a new Collection to availability? If so, what should the check be? Or is this not pertaining to availability (the collection I return)?

Reply:
> should I use [SOMETHING] to check if a certain condition is met before I assign a new Collection to availability

Yes.   Not necessarily just an IF statement though, you'd probably need more than an IF statement.  This test is for if Person A is booked 8:30-9:00, your result set should return open slots:  times before 8:30 and times after 9:00.